Dockerfile

```diff
@@ -1,16 +1,14 @@
-FROM amazonlinux:2
+FROM ubuntu:latest
```

Comment:
did we want to pin to a version or SHA?

Reply:
I will change it to 20.10.7 which is the current latest.

Reply:
I can only get ubuntu latests otherwise it seems to fail for various reasons.